lib/virt_autotest/utils.pm

```diff
@@ -147,4 +166,30 @@ sub upload_y2logs {
     save_screenshot;
 }
 
+sub ensure_default_net_is_active {
```

Comment:
This is redundant to [my current PR](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10937) and needs perhaps a refactoring after the first test runs.

Since I'm sure we have to change this after the first test runs anyway I suggest we leave it and refactor it when needed.

Reply:
No, this tests the network on the hypervisor. I'd say that the `ensure_running()` is redundant.

Reply:
I removed mine as your has all the coverage and more.